pkg/git/repowatch/getKey.go

```diff
@@ -20,11 +21,33 @@ import (
 	xssh "golang.org/x/crypto/ssh"
 )
 
-// getAuth tries to find an SSH authentication method.
+func isHttpRepo(repoURL string) bool {
+	parsedURL, err := url.Parse(repoURL)
+	if err != nil {
+		return false
+	}
+	if strings.HasPrefix(parsedURL.Scheme, "http") {
+		return true
+	}
+	return false
+}
+
```

Comment:
Please add documentation (perhaps move from `getAuthSSH`), since this is the entry point.